dev-tools/packaging/packages.yml

```diff
@@ -1007,6 +1013,19 @@ specs:
           '{{.BeatName}}{{.BinaryExt}}':
             source: ./build/golang-crossbuild/{{.BeatName}}-{{.GOOS}}-{{.Platform.Arch}}{{.BinaryExt}}
 
+    # Cloud specific docker image
+    - os: linux
+      arch: amd64
+      types: [docker]
+      spec:
+        <<: *agent_docker_spec
+        <<: *agent_docker_cloud_spec
+        <<: *elastic_docker_spec
+        <<: *elastic_license_for_binaries
```

Comment:
If I used a previous defined spec (like the complete variant), then docker images were not correctly generated:

```

REPOSITORY                                                     TAG                                              IMAGE ID       CREATED             SIZE
docker.elastic.co/beats/elastic-agent-cloud                    8.0.0                                            32ed8b53650d   2 minutes ago       843MB
docker.elastic.co/beats/elastic-agent-ubi8                     8.0.0                                            a5a7dee0d5f2   2 minutes ago       299MB
docker.elastic.co/beats/elastic-agent-cloud-complete           8.0.0                                            efa622cf7b28   12 minutes ago      1.88GB
```

so the `-cloud` prefix was added to the `complete` one.